src/site/styles/index.md-njk

```diff
@@ -0,0 +1,33 @@
+---
+layout: layouts/markdown-page-with-side-nav.njk
+title: Styles
+description: Make your service look and feel like other Defence services for internal users.
+is_root: true
+order: 2
+tags: styles
+---
+
+Make your service look and feel like other Defence services for internal users.
+
+If you are using MOD.UK Frontend in your build, the coded examples in the Design
+System will not need additional styling.
+
+If you need to apply styles manually, you should still follow existing
+conventions. For example, do not assign new meanings to colours, do not change
+the style of buttons or adjust the thickness of borders on form inputs.
+
+## Typography, spacing and layouts
+
+The [MOD.UK Figma Design Library]({{ vars.figmaLinks.designLibrary }}) shows how
+to apply:
+
+- [typography]({{vars.figmaLinks.styles}})
+- [spacing]({{vars.figmaLinks.styles}})
+- [layouts]({{vars.figmaLinks.styles}})
+
+This section has a guide for [applying MOD colours](/styles/colours/).
+
+If you cannot find what you need, contact the
+[user-centered design community](https://servicemanual.digital.mod.uk/your-community/user-centred-design/ "https://servicemanual.digital.mod.uk/your-community/user-centred-design/")
+or check the
+[GOV.UK Design System](https://design-system.service.gov.uk/ "https://design-system.service.gov.uk/").
```

Comment:
May want to remove the bits in quotes (ends up in the title attribute) as we haven't done that elsewhere (and I don't see it on GOV.UK sites).